llvm/lib/CodeGen/TargetLoweringObjectFileImpl.cpp

```diff
@@ -1194,18 +1194,34 @@ const MCExpr *TargetLoweringObjectFileELF::lowerRelativeReference(
       MCSymbolRefExpr::create(TM.getSymbol(RHS), getContext()), getContext());
```

Comment:
`lowerRelativeReference` uses MCSymbolRefExpr. It seems legacy (probably should be removed).